### PR TITLE
Adjusting scores

### PR DESCRIPTION
**Pull request by AnonymousVeNomicPlayer**
The update of the scores by Nils was not correct (he gave himself too few points). The rules do not make it completely clear what is supposed to happen in this case. This pull request adjusts the scores to the in my opinion correct number of points.
Josh suggested (and I would agree with him) that one should not receive points for not voting. Also it is not completely clear whether the number of points the proposer shall receive is obtained as number of positive votes / number of votes (with not sending an email not counting as a vote) or positive votes / number of eligible voters.
The proposed pull request
- awards no points for not voting
- calculates the number of points for the proposer as 4/5\*10, i.e. as number of positive votes/number of votes\*(rule number-291)

If you have a different opinion, comment on this pull request. If we do not agree in the end, we should let the Judiciary decide.

### CODE REVIEW

**Comment by malte-l** [comment]
I disagree with both proposed things, and I think the rules are actually quite clear on this. I think the misinterpretation (in my opinion) of the rules you made was in thinking that there was someone in the last turn that did not vote. In particularly the statement "not sending an email not counting as a vote" contradicts the wording and intention (when I wrote the first formulation) of the rule on voting. Reasoning below:

Leitsätze[1]: 

1. Rule 120 allows a valid vote to be cast by not emailing the proponent of the rule on voting day.
2. The rules currently prevent a situation from appearing, in which a player had violated rule 213.

A. Rule 120 says: `The procedure for voting on rules is as described below and must be followed`. Furthermore, the rule then **explicitly** does not require anyone to send an email: `Eligible voters **may** send an email` (my emphasis). As far as I know, everyone followed rule 120 in the last turn, including the people that did not send an email - as explicitly allowed - and have thus successfully voted on the rule, according to rule 120. (In particular, no one is in violation of rule 213, which was one of the motivations of having the `may` in there).
B. Rule 120 knows of two ways of voting: by sending an email referring to the voting of the turn (but only the first one counts) or not sending an email. For both of those, the third point specifies if the vote is to be interpreted as an approval or not. As the rules only know of "approved" and "not approved", this is the only distinction necessary.
C. The point whether the rules are to be interpreted as awarding points for "not voting" (itself violating rule 213) are thereby moot, as this can not happen.
D. Let me just add in the end that not awarding those 10 points for not sending an email would only have the effect that it makes sense for everyone to send up an automatic email containing anything other than an approval, to be sent eg. every day at 23:58 to everyones voting-email-address. Because then the player would not need to think about it, voting in the last 5 minutes is rare, and this gives the option of getting these 10 points. I see no advantage in this kind of spamming and no reason on the other hand to force people to send an email, when not sending one has no other effect on the game.

[1]: I would be interesting to know if there is an English word for that. But both the US and British supreme courts seem not to have Leitsätze, so perhaps there is no term for this.

**Comment by Snark1994** [comment]
Re _Leitsätze_: WordReference translates it as "Guiding principle". If I understand its meaning correctly, I would translate it as "Ruling" (i.e. an interpretation of the way that laws should be interpreted.)

Re **A**, **B**: I can find no evidence in Rule 120 that says not sending an email counts as voting: I think it is entirely silent on the matter. The closest I get is "...then the player's vote shall be treated as if no email had been sent". 

Re **C**: 213 is only about the proposer of the rule, and I don't think it is really relevant to the issue at hand, since it specifically states that "every player must **vote to approve** all rules that they propose", _i.e._ the question at hand of whether or not you can approve without voting is irrelevant.

Re **D**: I agree that that would be an unfortunate side-effect. However, I think incentivising people to vote (by which I mean, look at the rule, think about it, and then vote) is also a good thing.

In summary, I think the rules are silent on the issue, and would be happy for you to invoke judgement. I think the question at hand is "would you rather that we incentivise people to vote, or would you rather that we don't incentivise people to send automated emails until we introduce a rule to prohibit this?"

**Comment by Snark1994** [comment]
Oh I see, that makes more sense. In the light of that, I am happy not to merge the pull request.
